Components/Alert/alert.css

```diff
@@ -1,233 +1,241 @@
-@import url('/CSS/components.css');
+@import url("/CSS/components.css");
 
 /*********** alert ************/
 
 .alert--container {
-    margin: 10vh;
-    width: 70vw;
-    margin-top: 15vh;
+  width: 70vw;
+  margin: 10rem 0 0 32rem;
 }
 
 .alerts {
-    margin: 6vh 0 11vh 0;
+  margin: 3rem 0 5rem 0;
 }
 
 /* alerts */
 
 .alert {
-    height: 8vh;
-    width: 55vw;
-    font-size: 1.5rem;
-    margin: 2vh 0;
-    border-radius: 5px;
-    display: flex;
-    align-items: center;
-    box-shadow: 10px 10px 5px var(--boxShadow-color);
+  height: 8vh;
+  width: 40vw;
+  font-size: 1.5rem;
+  margin: 2rem 0;
+  border-radius: 5px;
+  display: flex;
+  align-items: center;
+  box-shadow: 10px 10px 5px var(--boxShadow-color);
 }
 
-.bold {
-    padding: 1rem;
+.bold-txt {
+  padding: 1rem;
 }
 
 /* simple alert */
 
-.btn__simple--primary,  .btn__outlined--primary{
-    padding: 1rem;
-    background-color: var(--blue-color);
-    color: var(--white-color);
-    border-radius: 5px;
+.btn__simple--primary,
+.btn__outlined--primary {
```

Comment:
create .btn__simple or .btn__outlined and then create separate classes like btn--primary that way you will reduce the no. of classes yu have to create

Reply:
right